process/indexer/operationHandler.go

```diff
@@ -0,0 +1,78 @@
+package indexer
+
+import (
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-core-go/data/outport"
+	"github.com/multiversx/mx-chain-core-go/marshal"
+	"github.com/multiversx/mx-chain-core-go/websocketOutportDriver/client"
+	"github.com/multiversx/mx-chain-core-go/websocketOutportDriver/data"
+	"github.com/multiversx/mx-chain-sovereign-notifier-go/process"
+)
+
+type operationHandler struct {
+	indexer           process.Indexer
+	marshaller        marshal.Marshalizer
+	operationHandlers map[data.OperationType]client.HandlerFunc
+}
+
+// NewOperationHandler creates a new operation handler
+func NewOperationHandler(indexer process.Indexer, marshaller marshal.Marshalizer) (client.OperationHandler, error) {
+	if check.IfNil(marshaller) {
+		return nil, errNilMarshaller
+	}
+	if check.IfNil(indexer) {
+		return nil, errNilIndexer
+	}
+
+	opHandler := &operationHandler{
+		indexer:    indexer,
+		marshaller: marshaller,
+	}
+
+	opHandler.operationHandlers = map[data.OperationType]client.HandlerFunc{
+		data.OperationSaveBlock:             opHandler.saveBlock,
+		data.OperationRevertIndexedBlock:    dummyHandler,
```

Comment:
are these meant to be dummy or it is only temporary? 
If not temporary, I would rename dummy to something like nopHandler (sounds more technical)

Reply:
Probably dummy like this forever, but possible to change
Right, good suggestion, renamed to `noOpHandler`